manim/mobject/mobject.py

```diff
@@ -1119,6 +1120,19 @@ def throw_error_if_no_points(self):
                       "for a Mobject with no points"
             caller_name = sys._getframe(1).f_code.co_name
             raise Exception(message.format(caller_name))
+    
+    # About z-index
+    def set_z_index(self, z_index_value):
+        self.z_index = z_index_value
+        return self
+    
+    def get_z_index(self):
```

Comment:
Getters are anti-pattern in python although manim is riddled with them [[1](https://docs.quantifiedcode.com/python-anti-patterns/correctness/implementing_java-style_getters_and_setters.html)]. set_z_index() above can stay since it allows for chaining.